build-system/compile/bundles.config.extensions.json

```diff
@@ -176,9 +176,7 @@
     "name": "amp-app-banner",
     "version": "1.0",
     "options": {
-      "hasCss": true,
```

Comment:
we're removing `app-banner` from this config, but we're not adding it to the `bundles.config.bento.json` file and we're not moving the files to the `src/bento/components` directory. Why are we removing these properties here?

Reply:
👏  This is a good catch. I had originally planned to move that component over in this PR but you informed me in an earlier conversation that it is actually not present in the [Bento AMP component tracker](https://docs.google.com/spreadsheets/d/1Yk3smB5ysHNFyZk67CDi-Zmkhomdoz7LnZhm7NsE2EQ/edit#gid=268848378) so it shouldn't be moved.